### PR TITLE
Add logging for creation of new runtimes and engines

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: src/backends/tensorrt/plan_backend.cc
Comment (deadeyegoodwin) [inline src/backends/tensorrt/plan_backend.cc:416]: Shouldn't all these logs be in LoadPlan where the engine and runtime are actually created?
Also, you should always print the gpu_device, even when dla is used.

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: src/backends/tensorrt/plan_backend.cc
Comment (CoderHam) [reply src/backends/tensorrt/plan_backend.cc:416]: The reason we cannot log this inside LoadPlan is because in the dynamic shape we destroy the engine after creation. 
We also do not pass the gpu_id, and model name to LoadPlan().

Comment (deadeyegoodwin) [reply src/backends/tensorrt/plan_backend.cc:416]: I see, then can just reduce it to a single line. Since it is verbose log it is ok to just always print gpu id and dla id even when dla id is -1

**Review by tanmayv25 — DISMISSED** [review]

**Review by tanmayv25 — APPROVED** [review]

**Comment by CoderHam** [comment]
@deadeyegoodwin I cannot merge until I get a re-review from you

**Pull request merged**
